### PR TITLE
v3 inconsistency fix

### PR DESCRIPTION
**Pull request by featalion**
Related to the issue #41. Please, review, test, etc.

### CODE REVIEW

File: IronMQ.class.php [hunk 1]
```diff
@@ -778,20 +780,25 @@ public function getMessagePushStatuses($queue_name, $message_id)
...
      *
      * @param string $queue_name
      * @param string $message_id
-     * @param string $subscription_id
+     * @param string $reservation_id
+     * @param string $subscriber_name
      * @return mixed
      */
-    public function deleteMessagePushStatus($queue_name, $message_id, $subscription_id)
+    public function deleteMessagePushStatus($queue_name, $message_id, $reservation_id, $subscriber_name)
```
Comment (featalion) [inline IronMQ.class.php]: I assume, that this method must be renamed to something like `deletePushMessage`.

Comment (thousandsofthem) [inline IronMQ.class.php]: Please do, `deletePushMessage ` sounds much better

File: IronMQ.class.php [hunk 2]
```diff
@@ -778,20 +780,25 @@ public function getMessagePushStatuses($queue_name, $message_id)
...
     {
+        $req = array(
+            'reservation_id': $reservation_id,
+            'subscriber_name': $subscriber_name
```
Comment (thousandsofthem) [inline IronMQ.class.php]: Please use old array syntax, php-5.3 still supported but can't use new array syntax

**Comment by featalion** [comment]
@thousandsofthem updated

**Comment by thousandsofthem** [comment]
@featalion Thanks! I'll merge & test & release

**Pull request merged**